class Solution {
    public int maxDepth(String s) {
        int depth=0;
        int maxDepth=-1;
        for(char ch:s.toCharArray()){
            if(ch=='('){
                depth++;
            }
            if(ch==')'){
                depth--;
            }
            maxDepth=Math.max(depth,maxDepth);
        }
        return maxDepth;
    }
}